src/Form/Type/OrderItemType.php

```diff
@@ -38,6 +40,11 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                 'class' => ProductVariant::class,
                 'label' => 'sylius.ui.variant',
             ])
+            ->add('customUnitPrice', MoneyType::class, [
+                'label' => 'sylius.ui.unit_price',
+                //temporary solution
```

Comment:
Is it still a temporary solution?

Reply:
Damn it, I forgot about that :/ Gonna fix it soon

Reply:
OK, I would vote for merging it and open an issue 😄 It will just take me some time to figure out what is wrong with my idea that should work (xD) and in the meanwhile it can be merged whatsoever.